### DISCUSSION TITLE
No prefix on exported APIs

### DISCUSSION THREAD

**Discussion by rchl** [intlify/vue-i18n-composable#12] [General]
Would it make sense to have those APIs named the same as in the Options API? I mean with the `$` prefix?

https://github.com/intlify/vue-i18n-composable/blob/a4579eb962a0374144114745963546610e945248/src/index.ts#L50-L54

**Comment by kazupon** [comment]
That good point!
`vue-i18n-composable` is vue-i18n Composition API for Vue 2.x.
So, We only provide a few APIs, but from now on, `vue-i18n-composable` will support to back-port from vue-i18n@v9.x

**Reply by rchl** [reply]
So what do you mean by that? That it's something that is worth adding or not?

That said, I guess vue-i18n exposes `$t` and co. in the template anyway so this is just a complementary API and maybe doesn't need to follow the same naming convention...

**Comment by kazupon** [comment]
I mean that the Composition API provided by vue-i18n@v9.x is back-ported to this package.
The APIs prefixed with `$` are not going to be exposed. We're not going to expose the API prefixed with `$`, because as you said, it's already provided in the options API
From now on, The reason for enhancing this package is to support users who are using Vue 2.x + `@vue/composition-api` to migrate to Vue3 + vue-i18n@v9.x using the Composition API. 2️⃣ ➡️ 3️⃣
